src/main/java/Duke.java

```diff
@@ -1,10 +1,227 @@
+        System.out.println("\t____________________________________________________________");
+    }
+
+    public static void storeText(String text) throws DukeException {
+        String task;
+        String at;
+        String by;
+
+        //Assign deadline task
+        if (text.startsWith("deadline")) {
+            task = text.substring(9, text.indexOf(" \\by"));
+            by = text.substring(text.indexOf("\\by") + 4, text.length());
+            taskList.add(new Deadline(task, by));
+            printTaskAssignment(taskList.size() - 1);
+
+        } else if (text.startsWith("event")) {
+            //Assign event task
+            task = text.substring(6, text.indexOf(" \\at"));
+            at = text.substring(text.indexOf("\\at") + 4, text.length());
+            taskList.add(new Event(task, at));
+            printTaskAssignment(taskList.size() - 1);
+
+        } else if (text.startsWith("todo")) {
+            //Assign todos task
+            task = text.replace("todo", "");
+            if (task.length() < 1) {
+                throw new StringIndexOutOfBoundsException();
+            }
+            taskList.add(new Todo(task));
+            printTaskAssignment(taskList.size() - 1);
+
+        } else {
+            throw new DukeException();
+        }
+    }
+
+    public static void displayList() {
+        int index = 1;
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Here are the tasks in your list:");
+        for (Task listTask : taskList) {
+            if (listTask == null) {
+                break;
+            }
+            System.out.println("\t " + index + "." + "[" + listTask.getTaskSymbol() + "]" + "["
+                    + listTask.getStatusIcon() + "] " + listTask);
+            index++;
+        }
+        System.out.println("\t____________________________________________________________");
+    }
+
+    public static void printTaskAssignment(int index) {
+        int taskIndex = index + 1;
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Got it. I've added this task:");
+        System.out.println("\t\t" + "[" + taskList.get(index).getTaskSymbol() + "]" + "[" +
+                taskList.get(index).getStatusIcon() + "]" + " " + taskList.get(index));
+        System.out.println("\t Now you have " + taskIndex + " tasks in the list.");
+        System.out.println("\t____________________________________________________________");
+    }
+
+/*    private static void printFileContents(String filePath) throws FileNotFoundException {
+        File f = new File(FILE_PATH);
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            System.out.println(s.nextLine());
+        }
+    }*/
+
+    public static void loadProgress() throws FileNotFoundException {
+        //File contains something
```

Comment:
I do not understand this comment.

Reply:
This comment should have been removed. Apologies for the confusion.